src/components/presentation/toDoList/toDos.js

```diff
@@ -12,11 +12,15 @@ class ToDos extends Component{
           </tr>
         </thead>
         <tbody>
-          {this.props.todos.map(todo => <ToDoContainer key={todo.id} {...todo} />)}
+          {this.props.toDos.ids.map(id => <ToDoContainer key={id} {...this.props.toDos.data[id]} />)}
```

Comment:
Yep thats right. You can go one step better after this and use a memoised selector for this https://github.com/reactjs/reselect#accessing-react-props-in-selectors